stdlib/ballerina-http/src/main/java/org/ballerinalang/net/http/HttpUtil.java

```diff
@@ -719,16 +708,17 @@ private static void extractBasicConfig(Annotation configInfo, Set<ListenerConfig
         }
     }
 
-    public static ChunkConfig getChunkConfig(String chunking) {
-        ChunkConfig chunkConfig;
-        if (HttpConstants.CHUNKING_AUTO.equalsIgnoreCase(chunking)) {
-            chunkConfig = ChunkConfig.AUTO;
-        } else if (HttpConstants.CHUNKING_ALWAYS.equalsIgnoreCase(chunking)) {
-            chunkConfig = ChunkConfig.ALWAYS;
-        } else if (HttpConstants.CHUNKING_NEVER.equalsIgnoreCase(chunking)) {
-            chunkConfig = ChunkConfig.NEVER;
+    public static ChunkKeepAliveConfig getKeepAliveChunkConfig(String keepAliveChunkConfig) {
```

Comment:
Please refer to my comment on mixing keep-alive configs and chunking configs in [https://github.com/wso2/transport-http/pull/60](https://github.com/wso2/transport-http/pull/60)